src/lib/gundb/UserStorageClass.js

```diff
@@ -1409,7 +1401,7 @@ export class UserStorage {
     let display
```

Comment:
```suggestion
    const { privateFieldDisplay } = this

    let display = privateFieldDisplay
```

Reply:
just initialize var with privateFieldDisplay value, then reassign in switch cases where it should have different. value